requirements.txt

```diff
@@ -0,0 +1 @@
+oras == 0.1.30
```

Comment:
If we are using embedded python in IPM, does this further limit the earliest IRIS version that IPM can be installed on?
I believe requirements.txt is currently always enforced. We should maybe make this optional so that IPM can still be installed on older versions of IRIS which don't have embedded python with just ORAS support disabled unless embedded python exists.

Reply:
Thanks for the feedback! Just had a discussion with @isc-tleavitt and it seems one possible approach is allow IRIS versions without embedded python to tolerate a failure to compile requirements.txt and handle run time issues by preventing the creation of ORAS repository definitions. This would also mean avoiding [Language = python ] methods so that the classes compile.

Reply:
@isc-shuliu @isc-tleavitt Not a blocker for this PR but what happens behind a firewall with installing python dependencies? Should IPM be packaged with a version of oras as a backup in case the internet is not available to install the latest version?
cc @isc-eneil @isc-jili @isc-jlechtne